lib/helpers.js

```diff
@@ -26,14 +26,21 @@ function formatTags(tags, telegraf) {
 function overrideTags (parent, child, telegraf) {
   var childCopy = {};
   var toAppend = [];
+  var duplicateKeys = [];
+
   formatTags(child, telegraf).forEach(function (tag) {
     var idx = typeof tag === 'string' ? tag.indexOf(':') : -1;
     if (idx < 1) { // Not found or first character
       toAppend.push(tag);
     } else {
-      childCopy[tag.substring(0, idx)] = tag.substring(idx + 1);
+      if(childCopy[tag.substring(0, idx)]) {
+        childCopy[tag.substring(0, idx)].push(tag.substring(idx + 1));
+      } else {
+        childCopy[tag.substring(0, idx)] = [tag.substring(idx + 1)];
+      }
```

Comment:
This kind of logic can be simplified by:

```js
const tagName = tag.substring(0, idx);
const tagValue = tag.substring(idx + 1);

childCopy[tagName] = childCopy[tagName] || [];
childCopy[tagName].push(tagValue);
```

Reply:
That's code from hot-shots. I was trying to keep the changes to the original code to a minimum.